### ISSUE TITLE
Tradução para o espanhol.

### ISSUE THREAD

**Issue by vidimus** [WagnerMD/MoMCenarios#1]
Olá, @WagnerMD , um amigo quer traduzir seu cenário para o espanhol. Eu gostaria de saber se ele pode e se você está disponível para atualizar o cenário quando a tradução estiver pronta.

**Comment by WagnerMD** [comment]
Boa tarde mestre, pode traduzir sim, fico muito honrado na verdade...

O Cenário contém um pequeno erro que identificamos após ser publicado... Eu
consegui corrigir, mas não consegui upar ele novamente...

Será que você conseguiria me ajudar?
Assim já fica zerado para quando for traduzido

Um forte abraço

Em sex, 6 de ago de 2021 14:15, vidimus ***@***.***> escreveu:

**Comment by vidimus** [comment]
Sim, claro, @WagnerMD qual é o erro? Para carregar a nova versão, só precisa de carregar os ficheiros com o mesmo nome no repositorio (upload files). São então substituídos e actualizados em valkyrie. É muito importante que os novos ficheiros tenhan exactamente o mesmo nome que a versão previa

**Comment by WagnerMD** [comment]
Perfeito Vidimus, me desculpa a demora em responder, estava bem ocupado
nesta ultima semana.
Vou baixar o Github novamente porque estou com outro PC e vou fazer estas
alterações.
AINDA ESTA semana te dou o OK de quando estiver atualizado :D

Em sex., 6 de ago. de 2021 às 19:57, vidimus ***@***.***>
escreveu:

**Comment by vidimus** [comment]
@WagnerMD 

Apenas o ficheiro com a tradução:
https://github.com/vidimus/Valkyrie-Mom-scenarios-translations/blob/master/Recuerdos%20corrompidos/Localization.Spanish.txt

O pacote enteiro, só para carregar, com a tradução já incluída:
https://github.com/vidimus/Valkyrie-Mom-scenarios-translations/tree/master/Recuerdos%20corrompidos%20-%20escenario%20completo

**Comment by WagnerMD** [comment]
Bom dia Vidimus, estou MUITO feliz!!
Vou pedir ajuda para os amigos da comunidade aqui do Brasil
Eu preciso de ajuda para atualizar, não estou conseguindo...

Me desculpe a inexperiência com o GitHub hehehe

Em sáb., 4 de set. de 2021 às 20:34, vidimus ***@***.***>
escreveu:

**Comment by WagnerMD** [comment]
ATUALIZADO!

Me ajudaram a atualizar os arquivos Vidimus.
Novamente, obrigado pela ajuda, qualquer duvida estou a disposição

Otima semana!

Em ter., 7 de set. de 2021 às 08:46, wagner dall agnol <
***@***.***> escreveu:
